publications/papers/optimal-sizing-paper/abstract.tex

```diff
@@ -0,0 +1,7 @@
+\begin{abstract}
```

Comment:
I hope this wasn't the abstract you included in your actual optimal sizing ANS summary submission... or is this a journal article version?... What am I looking at here?